docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```diff
@@ -0,0 +1,152 @@
+        <li> <code>exact_match_out</code>: if you set <code>exact_match_out=False</code>, then you don’t need to include all the elements of the result object - only the ones that are important to test, such as <inlineCode>{`{"success": True}`}</inlineCode>. </li>
+    </ul>
+</ul>
+</details>
+
+In our example, `data` will have two columns, "x" and "y", each with five rows. 
+If you define multiple columns, make sure that they have the same number of rows. 
+When possible, include test data and tests that includes null values (`None` in the Python test definition).
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_max_to_be_between_custom.py#L46
+```
+
+:::note
+When you define data in your examples, we will mostly guess the type of the columns. 
+Sometimes you need to specify the precise type of the columns for each backend. Then you use the `schema` attribute (on the same level as `data` and `tests` in the dictionary):
+
+```console
+"schemas": {
+  "spark": {
+    "x": "IntegerType",
+  },
+  "sqlite": {
+    "x": "INTEGER",
+  },
+```
+:::
+
+#### 3. Defining our tests
+
+In our example, `tests` will be a list containing dictionaries defining each test. 
+
+You will need to:
+1. Title your tests (`title`);
+2. Define the input for your tests (`in`);
+3. Decide how precisely you want to test the output of your tests (`exact_match_out`);
+4. Define the expected output for your tests (`out`).
+
+If you are interested in contributing your Custom Expectation back to Great Expectations, you will also need to decide if you want these tests publically displayed to demonstrate the functionality of your Custom Expectation (`include_in_gallery`).
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_max_to_be_between_custom.py#L44-L76
+```
+
+#### 4. Verifying our tests
+
+If you now run your file, `print_diagnostic_checklist` will attempt to execute these example cases.
+
+If the tests are correctly defined, and the rest of the logic in your Custom Expectation is already complete,
+you will see the following in your Diagnostic Checklist:
+
+```console
+✔ Has at least one positive and negative example case, and all test cases pass
+```
+
+<div style={{"text-align":"center"}}>
+<p style={{"color":"#8784FF","font-size":"1.4em"}}><b>
+Congratulations!<br/>&#127881; You've successfully created example cases & tests for a Custom Expectation! &#127881;
+</b></p>
+</div>
+
+#### 5. Contribution (Optional)
+
+This guide will leave you with test coverage sufficient for [contribution](../contributing/how_to_contribute_a_new_expectation_to_great_expectations.md) back to Great Expecations at an Experimental level;
+if you're interested in having your contribution accepted at a Beta level, these tests will need to pass for all supported backends (Pandas, Spark, & SQLAlchemy).
```

Comment:
```suggestion
This guide will leave you with test coverage sufficient for [contribution](../contributing/how_to_contribute_a_new_expectation_to_great_expectations.md) back to Great Expecations at an Experimental level.  

If you're interested in having your contribution accepted at a Beta level, these tests will need to pass for all supported backends (Pandas, Spark, & SQLAlchemy).
```
While a semicolon technically would work to separate these two sentences, it is more consistent to treat them as individual statements since the third scenario -- full acceptance into the Great Expectations codebase at Production level -- is an individual statement.  It is also generally easier for users to parse shorter sentences, so splitting this into two discrete statements increases readability.